hadoop-hdds/container-service/src/test/java/org/apache/hadoop/ozone/container/ozoneimpl/TestContainerReader.java

```diff
@@ -217,7 +217,7 @@ private List<Long> addBlocks(KeyValueContainer keyValueContainer,
   @Test
   public void testContainerReader() throws Exception {
```

Comment:
How about we simply add a test which should simply create Recovering container and restart the DN and check?

Refer: TestContainerCommandsEC for creating recovering containers.